### ISSUE TITLE
Display Radio Covers as squares

### ISSUE THREAD

**Issue by bascht** [gnumdk/lollypop#227]
Well, at first: Thanks for creating Lollypop – it's by far my favourite player on the GNOME desktop so far. :+1: 

One tiny thing is bugging me: Radio stations aren't displayed in a squared grid, like the covers – which results in a slightly infuriating sight (including the transparent backgrounds): :smiley_cat: 

![screenshot from 2015-08-10 11-31-52](https://cloud.githubusercontent.com/assets/51264/9166878/c38fbd8e-3f53-11e5-8445-86908ba71af5.png)
Running Lollypop 0.9.42

**Comment by gnumdk** [comment]
What distribution? Here radios are shown as square :-(

**Comment by gnumdk** [comment]
Ok, bug found :)

**Comment by bascht** [comment]
Well, that was quicker than I could've replied. :D It's the one from the Fedora System repo. :)

**Comment by gnumdk** [comment]
But don't know how to fix it for now

**Comment by gnumdk** [comment]
I really find your covers enormous in your screenshot!

**Comment by bascht** [comment]
:blush: I'll take that as a compliment, but those are the sizes Lollypop suggests:
![out](https://cloud.githubusercontent.com/assets/51264/9176912/465deb7a-3f8e-11e5-8fc3-07a12be1d611.gif)

**Comment by gnumdk** [comment]
$ rm -rf .cache/lollypop

**Comment by gnumdk** [comment]
For information, what are your hidpi scale factor?

**Comment by bascht** [comment]
@gnumdk Awesome! I've just built a4bab15 and it's fine now! Thanks for the really quick fix. I was on hidpi scale 1 when I took the screenshot (external monitor) – but even hidpi 2x scaling is now properly scaled + squared:
![screenshot from 2015-08-10 23-32-12](https://cloud.githubusercontent.com/assets/51264/9184853/21c6f0b6-3fb8-11e5-9a8e-fcdc82f58a3a.png)

Fun fact: All of my music covers where *huge* but I didn't notice it that much, as they where all properly aligned. :) They are normal sized now: 
![screenshot from 2015-08-10 23-37-24](https://cloud.githubusercontent.com/assets/51264/9184968/cae8ada6-3fb8-11e5-9e69-dac407f1b013.png)

Thanks again! :+1:
